apps/vessel-history/features/dataviews/dataviews.slice.ts

```diff
@@ -1,19 +1,25 @@
 import { createAsyncThunk, createSelector } from '@reduxjs/toolkit'
 import { memoize, uniqBy } from 'lodash'
+import { stringify } from 'qs'
```

Comment:
is this different from JSON.stringify? @rdgfuentes

Reply:
Yes @fertandil87, it's the `stringify` function from `qs` npm package (https://github.com/ljharb/qs#stringifying)